### PR TITLE
Fix/remove datepicker and replace with dropdowns for year and month. 

### PR DESCRIPTION
**Pull request by orjwan98**
References issue #223 

What I did in this PR:

- [x] Remove date picker module from package.json
- [x] Re-implement date as two dropdowns
- [x] Year is required, month is optional
- [x] Add content that tells user to use the year that the event began at if it is something that spans across times.

### CODE REVIEW

File: client/src/Components/Form/Details/index.js [hunk 1]
```diff
@@ -66,20 +84,50 @@ const Details = ({
...
-          className={classes.selectLocation}
+          className={classes.selectDropdown}
+        >
+          {renderSelectItems(locations)}
+        </Select>
+      </FormControl>
+      {DateQuestion}
+      <FormControl className={classes.formControlDate}>
+        <InputLabel htmlFor="month" className={classes.label}>
+          Month (Optional)
```
Comment (Oliversw) [inline client/src/Components/Form/Details/index.js]: Okay I know this is annoyingly small, but I think ```Optional``` should be a lowercase ```optional``` just to match everything else on the page. Sorrryy but thank you!

File: client/src/Components/Form/Details/index.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React, { Fragment } from 'react';
-import DateFun from './DatePicker';
-import { locations } from '../../data.json';
-import ErrMsg from './Details.style';
+import { months, locations } from '../../data.json';
+import { ErrMsg, BoldText } from './Details.style';
```
Comment (Oliversw) [inline client/src/Components/Form/Details/index.js]: Eslint says that ```'BoldText' is declared but its value is never read.```

File: client/src/Components/Form/Details/index.js [hunk 3]
```diff
@@ -66,20 +84,50 @@ const Details = ({
...
+            name: 'month',
+            id: 'month',
+          }}
+          aria-label="month"
+          value={month}
+          onChange={dropdownChange}
+          disableUnderline
+          className={classes.selectDropdown}
+        >
+          {renderSelectItems(months)}
```
Comment (Oliversw) [inline client/src/Components/Form/Details/index.js]: This is probably one of the only dropdowns that shouldn't be sorted alphabetically :)

Screen Shot 2019-05-29 at 3 34 23 PM
😂

**Comment by orjwan98** [comment]
@teenie-quaggard and @Oliversw changed, please review again :)

**Comment by teenie-quaggard** [comment]
Aside from merge conflicts, looks good to me!

**Pull request merged**